Dockerfile

```diff
@@ -36,6 +36,7 @@ WORKDIR /
 COPY --from=builder /istio-build/manager .
 COPY --from=builder /istio-build/istio_install .
 
+ENV GODEBUG="fips140=only,tlsmlkem=0"
```

Comment:
I suggest that we add that environment variable to the chart as well even though it is not explicitly needed